go-controller/pkg/node/heartbeat_dpu.go

```diff
@@ -25,7 +25,7 @@ const (
 	// This the default value for corev1.Node leases
 	defaultLeaseDurationSeconds = 40
 	// defaultLeaseNS is the default namespace for the lease.
-	defaultLeaseNS = "dpu-node-lease"
+	defaultLeaseNS = "ovn-kubernetes"
```

Comment:
nit: I would avoid a default here and instead make it required, otherwise I would avoid defaulting it in ovnkube.sh

Reply:
I will remove the default in the script. I need it here otherwise I'll break all the unit tests.